modules/ossm-rn-new-features.adoc

```diff
@@ -15,6 +15,193 @@ Module included in the following assemblies:
 
 This release adds improvements related to the following components and concepts.
 
+//Includes 2.6, 2.5.3, 2.4.9.
+
+//OSSM 2.6 Release Day tentatively scheduled for Wednesday, July 10, 2024 --> delayed. New estimated release date: July 24, 2024.
+
+//This release will include new features such as OTEL+Tempo, Gateway API, and perhaps some others.
+
+[id="new-features-ossm-2-6"]
```

Comment:
Headings in modules (throughout the PR, not just here) should have a `_{context}` in their ID:
```suggestion
[id="new-features-ossm-2-6_{context}"]
```